pkg/packagekit/updates.jsx

```diff
@@ -592,6 +621,19 @@ class OsUpdates extends React.Component {
             .fail(this.handleLoadError);
     }
 
+    handleRestart() {
+        this.setState({state: "restart"})
+        // give the user a chance to actually read the message
+        window.setTimeout(() => {
+            cockpit.spawn(["shutdown", "--reboot", "now"], {superuser: true, err: "message"})
+                .fail(ex => {
+                    // Hmm - this is a Should Not Happen™, but at least show something
```

Comment:
You mean commenting out the error handling for shutdown? TBH I'd rather leave it in - if calling shutdown fails, then the "Restarting.." spinner will go on forever without any feedback.